sdk/spring/azure-spring-integration-servicebus/src/main/java/com/azure/spring/integration/servicebus/converter/ServiceBusMessageHeaders.java

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.integration.servicebus.converter;
+
+import com.azure.spring.integration.core.AzureHeaders;
+
+/**
+ * Azure service bus internal headers.
```

Comment:
not sure if this **internal** will cause confusion or not

Reply:
@stliu . Updated.